src/bindist/autodos7.bat

```diff
@@ -0,0 +1,6 @@
+@ECHO Off
+PATH D:\OPENDOS;D:\EMUBIN;D:\
+VERIFY OFF
+PROMPT [OpenDOS] $P$G
+SET OPENDOSCFG=D:\OPENDOS
+CALL D:\EMUBIN\UXMACROS.BAT
```

Comment:
This happens with every dos version. To reproduce, just create an empty batch file with only newlines in it. Then run this batchfile. You'll see as many C:\\> as you put newlines in that batchfile.

I just pushed a new version of the commit which add the ems driver and cdrom support btw. The included uxmacros doesn't work with FreeDOS and MS-DOS.